ocs_ci/deployment/deployment.py

```diff
@@ -259,6 +259,15 @@ def deploy_ocs_via_operator(self):
             'storage'
         ] = f"{device_size}Gi"
 
+        # Allow lower instance requests and limits for OCS deployment
+        if config.DEPLOYMENT.get('allow_lower_instance_requirements'):
+            deviceset_data["resources"] = {
+                {
+                    resource: {'Requests': None, 'Limits': None} for resource
+                    in ['mon', 'mds', 'rgw', 'mgr', 'noobaa']
+                }
+            }
```

Comment:
since deviceset_data["resources"] contains empty dictionary , we don't need to assign dictionary again
change to below:

deviceset_data["resources"] = {
                    resource: {'Requests': None, 'Limits': None} for resource
                    in ['mon', 'mds', 'rgw', 'mgr', 'noobaa']
                }